cwf/gateway/docker/docker-compose.yml

```diff
@@ -187,7 +187,7 @@ services:
     network_mode: host
     restart: always
     command: >
-      /bin/bash -c "envsubst < /etc/magma/templates/radius.conf.template > /var/opt/magma/tmp/radius.config.json &&
+      /bin/bash -c "envsubst < /etc/magma/templates/radius.analytics_disabled.conf.template > /var/opt/magma/tmp/radius.config.json &&
```

Comment:
so in the future is we want to move from analytics enabled and disabled we just need to switch it here,. The docker image will contain both files. correct? Or are you going to delete any of the files in the future?

Reply:
Yeah you're correct. I plan to keep both for now in case we need analytics at some point.